hw/xbox/mcpx_apu.c

```diff
@@ -471,9 +502,31 @@ static void gp_write(void *opaque, hwaddr addr,
 {
     MCPXAPUState *d = opaque;
 
+    assert(size == 4);
+
     MCPX_DPRINTF("mcpx apu GP: [0x%llx] = 0x%llx\n", addr, val);
 
     switch (addr) {
+    case NV_PAPU_GPXMEM ... NV_PAPU_GPXMEM + 0xFFF*4: {
```

Comment:
I'd prefer if you defined the different memregion sizes/offsets as constants.

Reply:
I'm following the style that has been established in the existing code.

I suggest we revise this when addressing what I described in "EP and GP use the same memory map in XQEMU": if we split EP and GP config, we will also define constants for their respective size probably.